### ISSUE TITLE
RSS feed

### ISSUE THREAD

**Issue by maxoriggi** [SachaG/SidebarFeedback#15]
Great site, but it really really needs an RSS feed.

**Comment by SachaG** [comment]
It has one actually: http://sidebar.io/feed.xml (right now it shows all links, not just the featured ones, but I'll fix that soon).

**Comment by maxoriggi** [comment]
Yep, great... but it should be linked in the head section so that it can be discovered by readers (both apps and real people 😄 )

**Comment by SachaG** [comment]
Good point :)
ᐧ

**Comment by maxoriggi** [comment]
I do love RSS, without it following a couple hundred sites would be impossible.
I know most people do not even know what it is, but given the target of your site, i think it's important.

**Comment by pribeiro78** [comment]
RSS feed is not a valid one so I can't see the feeds in my mac Notification Center.
https://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Fsidebar.io%2Ffeed.xml

**Comment by SachaG** [comment]
Thanks, I'll look into this.

**Comment by hail2u** [comment]
This invalid Sidebar feed cannot process with IFTTT or similar webservices that cannot monkey-patch by a user.

The problem is embedded images. You should include images into `description` element with `<img>` instead of `imageUrl` and `content` element (these elements are not defiend in RSS 2.0 spec and referenced XML modules’ spec).
